audit-library/src/main/java/com/pinterest/singer/loggingaudit/client/AuditHeadersGenerator.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2019 Pinterest, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.pinterest.singer.loggingaudit.client;
+
+import com.pinterest.singer.loggingaudit.client.utils.AuditHeadersUtil;
+import com.pinterest.singer.loggingaudit.thrift.LoggingAuditHeaders;
+
+public class AuditHeadersGenerator {
```

Comment:
add some comments to explain the motivation behind `AuditHeadersGenerator`?

Reply:
added comments in updated PR.